docker/build_jakartaeetck.sh

```diff
@@ -75,40 +70,34 @@ ant -version
 which java
 java -version
 
-export ANT_OPTS="-Xmx2G -Djava.endorsed.dirs=${GF_HOME}/$GF_TOPLEVEL_DIR/glassfish/modules/endorsed \
+export ANT_OPTS="-Xmx2G -Djava.endorsed.dirs=${JAKARTA_JARS}/endorsed \
                  -Djavax.xml.accessExternalStylesheet=all \
                  -Djavax.xml.accessExternalSchema=all \
 		 -DenableExternalEntityProcessing=true \
                  -Djavax.xml.accessExternalDTD=file,http"
 
 echo ########## Remove hard-coded paths from install/jakartaee/bin/ts.jte ##########"
-sed -e "s#^javaee.home=.*#javaee.home=$GF_HOME/$GF_TOPLEVEL_DIR/glassfish#g" \
-    -e "s#^javaee.home.ri=.*#javaee.home.ri=$GF_HOME/$GF_TOPLEVEL_DIR/glassfish#g" \
+sed -e "s#^javaee.home=.*#javaee.home=$JAKARTA_JARS#g" \
+    -e "s#^javaee.home.ri=.*#javaee.home.ri=$JAKARTA_JARS#g" \
     -e "s#^report.dir=.*#report.dir=$BASEDIR/JTReport#g" \
     -e "s#^work.dir=.*#work.dir=$BASEDIR/JTWork#g" $BASEDIR/install/jakartaee/bin/ts.jte > $BASEDIR/install/jakartaee/bin/ts.jte.new
 mv $BASEDIR/install/jakartaee/bin/ts.jte.new $BASEDIR/install/jakartaee/bin/ts.jte
 
 #tools.jar from jdk8 has old apis
-sed -i -e 's#tools\.jar=.*#tools.jar='${GF_HOME//\//\\\/}'\/glassfish6\/glassfish\/modules\/webservices-tools.jar:'${GF_HOME//\//\\\/}'\/glassfish6\/glassfish\/modules\/webservices-api.jar#g' $BASEDIR/install/jakartaee/bin/ts.jte
+sed -i -e 's#tools\.jar=.*#tools.jar='${JAKARTA_JARS//\//\\\/}'\/modules\/webservices-tools.jar:'${JAKARTA_JARS//\//\\\/}'\/modules\/webservices-api.jar#g' $BASEDIR/install/jakartaee/bin/ts.jte
 
 echo "Contents of modified TS.JTE file"
 cat $BASEDIR/install/jakartaee/bin/ts.jte
 
 echo "########## Trunk.Install.V5 Config ##########"
 cd $BASEDIR
-if [ -z "$GF_BUNDLE_URL" ]; then
-  echo "Using default url for GF bundle: $DEFAULT_GF_BUNDLE_URL"
-  export GF_BUNDLE_URL=$DEFAULT_GF_BUNDLE_URL
-fi
-wget --progress=bar:force --no-cache $GF_BUNDLE_URL -O latest-glassfish.zip
-unzip -q -o latest-glassfish.zip
-ls -l $GF_HOME/$GF_TOPLEVEL_DIR/glassfish/
 
+mkdir -p $JAKARTA_JARS/modules
+mkdir -p $JAKARTA_JARS/endorsed
 
-if [ ! -z "$GF_VERSION_URL" ]; then
-  wget --progress=bar:force --no-cache $GF_VERSION_URL -O glassfish.version
-  cat glassfish.version
-fi
+mvn -f $BASEDIR/docker/pom.xml dependency:copy-dependencies -DoutputDirectory="${JAKARTA_JARS}/modules" -Dmdep.stripVersion=true
```

Comment:
+1000 for using Maven for downloading the SPEC API dependencies which [sort of was brought up by Bill before](https://github.com/eclipse-ee4j/jakartaee-tck/issues/207#issuecomment-617482697).